docs/other_usages/replacing_file.md

```diff
@@ -0,0 +1,49 @@
+Inject files from other sources
+==================
```

Comment:
This line should have the same length as the title. IMHO it's better to use the alternate form, with a `#` on the beginning of the first line instead, to avoid additional changes in the future